serverless/guest/memory.go

```diff
@@ -0,0 +1,50 @@
+// guest wasm application programming interface for guest module
+package guest
+
+import (
+	"unsafe"
+)
+
+var (
+	// ReadBuf is a buffer used to read data from the host.
+	ReadBuf = make([]byte, ReadBufSize)
+	// ReadBufPtr is a pointer to ReadBuf.
+	ReadBufPtr = uintptr(unsafe.Pointer(&ReadBuf[0]))
+	// ReadBufSize is the size of ReadBuf
+	ReadBufSize = uint32(2048)
+)
+
+func GetString(fn func(ptr uintptr, size uint32) (len uint32)) (result string) {
```

Comment:
`GetString` could be implemented by wrapping `GetBytes`; besides, it seems to be unused?

Reply:
Yeah, it's not in use right now.